plugins/module_utils/common/ipaddress_utils.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
this can go into the plugin_utils directory not module utils since it's here for non-modules